src/.env

```diff
@@ -0,0 +1 @@
+MONGO_URI="mongodb+srv://vassbootcamp:LympI4tnrQeicK5B@task-manager.zok3csm.mongodb.net/?retryWrites=true&w=majority&appName=task-manager"
```

Comment:
you should keep the env variables on your local machine as it may contain sensitive information, just good practice to keep it locally